src/components/DisplayedDocument.ts

```diff
@@ -199,9 +199,7 @@ function DisplayedDocument(): m.Component<Attrs> {
 
             childIds.forEach((childId) => {
                 calculateDimensions(allDimensions, childId);
-                totalChildrenHeight += safeGetDimensions(
-                    allDimensions, childId,
-                ).heightWithChildren;
+                totalChildrenHeight += safeGetDimensions(allDimensions, childId).heightWithChildren;
```

Comment:
I'm curious -- was this an editor auto format thing, or a personal preference re: line length?

Reply:
Ah, yeah, this was just my editor enforcing the eslint rules configured in `.eslintrc.js`. Looks like this one came from `'airbnb-base'` (`function-paren-newline` rule).

Reply:
Ah, looks like my local eslint config was still broken. Fixed now, and I see that :+1: